src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <section class="calendar calendar--start-day-mon calendar--month-length-31">
+        <div class="calendar__day calendar__day--day-1"></div>
+        <div class="calendar__day calendar__day--day-2"></div>
```

Comment:
u don't need another modificators for your div's, the class `calendar__day` only should be enough. Then u can choose your day using the [nth-child()](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) pseudo-class in css.